### PR TITLE
misc: multi-network support

### PR DESCRIPTION
**Pull request by kilpatty**
Depends on: 
- [x] https://github.com/UrkelLabs/ucl/pull/88

Fixes: #15

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -38,11 +37,16 @@ import NotFoundScreen from "screens/errors/NotFound";
 // Internationalization
 import "../i18n/i18n";
 
+// Hooks
+import useNetwork from "hooks/useNetwork";
+
+//@todo useNetwork custom hook
```
Comment (kilpatty) [inline src/components/App.js]: Will remove this :/

File: src/screens/Blocks.js [hunk 2]
```diff
@@ -1,22 +1,29 @@
 import React, { Suspense } from "react";
-import { usePage } from "@urkellabs/ucl";
+import { usePage, useQuery } from "@urkellabs/ucl";
 
 // Resources
-import { useResource, useResultCache } from "rest-hooks";
-import BlockResource from "resources/BlockResource";
+// import { useResource, useResultCache } from "rest-hooks";
+// import BlockResource from "resources/BlockResource";
```
Comment (kilpatty) [inline src/screens/Blocks.js]: Need to delete.

File: src/screens/Blocks.js [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
+  // const blocks = useResource(BlockResource.listShape(), { offset: pageOffset });
+  // const { limit, total } = useResultCache(BlockResource.listShape(), {
+  //   offset: pageOffset
+  // });
+  // const params = useMemo(() => ({ p: page }), [page]);
 
-  return <BlocksTable blocks={blocks} pages={pages} page={page} />;
+  //@todo build out support for p in hnscan-backend
+  // const { data } = useQuery("/blocks/", { p: page });
+  // @todo make this a custom hook.
```
Comment (kilpatty) [inline src/screens/Blocks.js]: Delete most of the above.

File: src/components/shared/NetworkSwitcher.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      defaultValue={
+        options.find(element => element.value === network) || options[0]
+      }
+      onChange={kv => {
+        setNetwork(kv.value);
+      }}
+    />
+  );
+}
+
```
Comment (JGeiser9) [inline src/components/shared/NetworkSwitcher.js]: this all looks good and love the fact that this is getting implemented

File: src/containers/SearchBar.js [hunk 5]
```diff
@@ -97,6 +97,7 @@ const SearchBar = props => {
   //Search API Call
   const search = async e => {
     e.preventDefault();
+    //@todo convert this to useQuery
```
Comment (JGeiser9) [inline src/containers/SearchBar.js]: were you planning to do this in this PR?

File: src/screens/Home.js [hunk 6]
```diff
@@ -39,24 +34,22 @@ const IndividualCardContainer = styled.div`
 
 //@todo skeleton.
 function HomeView() {
-  //API Calls
-  const [summary, blocks, txs] = useResource(
-    [NetworkResource.detailShape(), {}],
-    [BlockResource.listShape(), { limit: 5 }],
-    [TransactionResource.listShape(), { limit: 5 }]
-  );
+  const { data: summary } = useQuery("/summary");
```
Comment (JGeiser9) [inline src/screens/Home.js]: this looks so much cleaner!

**Pull request merged**